### PR TITLE
Rename .md & replace "Apollo" with "meteor" in CoC

### PR DESCRIPTION
**Pull request by hwillson**
Just a few small tweaks to replace the Apollo references with Meteor. Thanks!

### CODE REVIEW

File: code-of-conduct.md [hunk 1]
```diff
@@ -8,11 +8,11 @@ The project gains strength from a diversity of backgrounds and perspectives in o
...
+This code of conduct applies to all spaces managed by the Meteor project or company. This includes Slack, GitHub issues, and any other forums created by the Meteor team which the community uses for communication. Breaches of this code outside these spaces may affect a person's ability to participate within them. We expect it to be honored by everyone who represents or participates in the project, whether officially or informally.
 
-If you believe someone is violating the code of conduct, please report it by emailing [community@apollodata.com](mailto:community@apollostack.com). 
+If you believe someone is violating the code of conduct, please report it by emailing [community@meteor.com](mailto:community@meteor.com).
```
Comment (hwillson) [inline code-of-conduct.md]: @abernix @benjamn Is this a valid email address? I just guessed, so let me know if it should be changed to something else.

**Comment by abernix** [comment]
As this file introduced another naming convention format, this seems like a good time to discuss.

I believe many projects use all capital, snake-cased (e.g. `CODE_OF_CONDUCT.md`, `CONTRIBUTING.md`).  Including, for example:

* https://github.com/babel/babel
* https://github.com/nodejs/node
* https://github.com/eslint/eslint
* https://github.com/graphql/graphql-js/
* https://github.com/facebook/jest
* https://github.com/facebook/react
* https://github.com/Microsoft/vscode
* https://github.com/npm/npm/

Maybe others... this leaves us with some considerations for existing files like `README.md`, which I think _must_ be capitalized (at least it used to be for `npm`) and `History.md` (which if we renamed could break links, though is typically called `CHANGELOG.md`).

File: code-of-conduct.md (hunk 1 above)
Comment (abernix) [reply code-of-conduct.md]: I think it is, but I will check. :)

**Comment by hwillson** [comment]
I noticed that as well - bringing everything in-line using capital snake-case gets my vote!

- `CODE_OF_CONDUCT.md`
- `CONTRIBUTING.md`
- `DEVELOPMENT.md`
- `CHANGELOG.md` (switched from `History.md`)
- `ISSUE_TRIAGE.md`
- `README.md`
- `ROADMAP.md`

If you agree, I can make these changes here (and fix any broken `History.md` links located in this repo).

File: code-of-conduct.md (hunk 1 above)
Comment (abernix) [reply code-of-conduct.md]: Confirmed, good to use community@meteor.com!

**Comment by hwillson** [comment]
Quick re-cap of the decision from our triage call - the following will be renamed in capital snake-case (since they're more policy related):

- CODE_OF_CONDUCT.md
- CONTRIBUTING.md
- DEVELOPMENT.md
- ISSUE_TRIAGE.md

Everything else will stay as is for now. Changes coming shortly - thanks!

**Comment by hwillson** [comment]
Filename changes are complete, so we should be all set here. Thanks!

**Comment by hwillson** [comment]
And just to add - once this is merged we should also merge https://github.com/meteor/meteor-feature-requests/pull/197.

**Pull request merged**